### PR TITLE
fix: Map ORM errors to internalServerError

### PR DESCRIPTION
**Pull request by EnriqueL8**
## Description
Mapping the ORM errors to internalServerError to not expose sensitive information about the database

### CODE REVIEW

File: Sources/Kitura/CodableRouter.swift [hunk 1]
```diff
@@ -434,6 +434,8 @@ public struct CodableHelpers {
      *            if valid, or HTTPStatusCode.unknown otherwise
      */
     public static func httpStatusCode(from error: RequestError) -> HTTPStatusCode {
+        // ORM status code 7XX mapped to internalServerError 500
+        if error.httpCode >= 700 { return HTTPStatusCode(rawValue: 500) ?? HTTPStatusCode.unknown }
```
Comment (djones6) [inline Sources/Kitura/CodableRouter.swift]: Do we want to limit the replacement to strictly the 7xx range,  ie. `>= 700` and `< 800`?

Comment (EnriqueL8) [reply Sources/Kitura/CodableRouter.swift]: yes

File: Sources/Kitura/CodableRouter.swift [hunk 2]
```diff
@@ -434,6 +434,8 @@ public struct CodableHelpers {
      *            if valid, or HTTPStatusCode.unknown otherwise
      */
     public static func httpStatusCode(from error: RequestError) -> HTTPStatusCode {
+        // ORM status code 7XX mapped to internalServerError 500
+        if error.httpCode >= 700 && error.httpCode < 800 { return HTTPStatusCode(rawValue: 500) ?? HTTPStatusCode.unknown }
```
Comment (fierydrake) [inline Sources/Kitura/CodableRouter.swift]: Should just `return .internalServerError`

Comment (tunniclm) [inline Sources/Kitura/CodableRouter.swift]: Should just `return .internalServerError`

Comment (EnriqueL8) [reply Sources/Kitura/CodableRouter.swift]: could do

Comment (EnriqueL8) [reply Sources/Kitura/CodableRouter.swift]: could no use `.internalServerError`... had to use `HTTPStatudCode.internalServerError`... it should be the same thing

**Comment by EnriqueL8** [comment]
Tests passed, but github not picking them up...okay to merge?

File: Sources/Kitura/CodableRouter.swift [hunk 3]
```diff
@@ -434,6 +434,8 @@ public struct CodableHelpers {
      *            if valid, or HTTPStatusCode.unknown otherwise
      */
     public static func httpStatusCode(from error: RequestError) -> HTTPStatusCode {
+        // ORM status code 7XX mapped to internalServerError 500
+        if error.httpCode >= 700 && error.httpCode < 800 { return HTTPStatusCode.internalServerError }
```
Comment (djones6) [inline Sources/Kitura/CodableRouter.swift]: This looks much nicer now :+1:

**Pull request merged**